### PR TITLE
docs(styles): update the documentation for Date Picker

### PR DESCRIPTION
**Pull request by ynnenu**
## Description
Updated the documentation of Date picker.

### CODE REVIEW

File: stories/datepicker/datepicker.stories.js [hunk 1]
```diff
@@ -424,7 +429,12 @@ export const DefaultAndCompactSizes = () => `<div style="display:flex;justify-co
 
 DefaultAndCompactSizes.parameters = {
     docs: {
-        iframeHeight: 500
+        iframeHeight: 500,
+        description: {
+            story: 'The default date picker is represented on the left part of the container in the example below. The default version of date picker is automatically generated through `fd-docs-date-picker` class. In order to generate a compact version of the date picker, add `fd-input--compact` in the input class.'
+
```
Comment (InnaAtanasova) [inline stories/datepicker/datepicker.stories.js:435]: Here you need to modify a little bit the text. You can start with "The date picker provides responsive behavior that allows for simple operation on all devices. It is smaller in compact mode and provides a touch-friendly size in cozy mode." This I took from Responsiveness section in https://experience.sap.com/fiori-design-web/date-picker/
We don't mention the modifier classes for compact as they are many and applied on every element that is a building block of the Date Picker. 
OR :) 
you can add something like (just with the correct English :D )
"For compact mode add the appropriate modifier classes, such as `fd-input--compact`, `fd-button--compact`, `fd-calendar--compact`, etc to the building blocks of the component."

File: stories/datepicker/datepicker.stories.js [hunk 2]
```diff
@@ -1094,7 +1104,12 @@ export const MobileLandscape = () => `<div class="fd-dialog-docs-static fd-calen
 MobileLandscape.storyName = 'Mobile landscape mode with today navigation';
 MobileLandscape.parameters = {
     docs: {
-        iframeHeight: 400
+        iframeHeight: 400,
+        description: {
+            story: 'The additional today button displayed in the navigation part of the calendar, sets the focus to the current date. The landscape sizing and roation of  the date picker is done by adding `fd-calendar--mobile-landscape` class.'
+
```
Comment (InnaAtanasova) [inline stories/datepicker/datepicker.stories.js:1110]: This is perfect!

File: stories/datepicker/datepicker.stories.js [hunk 3]
```diff
@@ -1327,6 +1342,11 @@ export const MobilePortrait = () => `<div class="fd-dialog-docs-static fd-calend
 MobilePortrait.storyName = 'Mobile portrait mode with today navigation';
 MobilePortrait.parameters = {
     docs: {
-        iframeHeight: 500
+        iframeHeight: 500,
+        description: {
+            story: 'Similar to the mobile landscape example, mobile portrait mode has a Today button. To make date picker visible in portrait mode please add `fd-calendar--mobile-portrait` class.'
+
```
Comment (InnaAtanasova) [inline stories/datepicker/datepicker.stories.js:1348]: This one, too

Comment (InnaAtanasova) [reply stories/datepicker/datepicker.stories.js:1348]: You don't need the `please add`, just `add`

**Review by InnaAtanasova — COMMENTED** [review]
Great job! Added a few comments. Let me know what you think and we can have a call to discuss it.

File: stories/datepicker/datepicker.stories.js [hunk 4]
```diff
@@ -1094,7 +1104,12 @@ export const MobileLandscape = () => `<div class="fd-dialog-docs-static fd-calen
 MobileLandscape.storyName = 'Mobile landscape mode with today navigation';
 MobileLandscape.parameters = {
     docs: {
-        iframeHeight: 400
+        iframeHeight: 400,
+        description: {
+            story: 'The additional today button displayed in the navigation part of the calendar, sets the focus to the current date. The landscape sizing and roation of  the date picker is done by adding `fd-calendar--mobile-landscape` class.'
```
Comment (mikerodonnell89) [inline stories/datepicker/datepicker.stories.js:1109]:
```suggestion
            story: 'The additional "today" button displayed in the navigation part of the calendar sets the focus to the current date. The landscape sizing and rotation of the date picker is done by adding the `fd-calendar--mobile-landscape` class.'
```

File: stories/datepicker/datepicker.stories.js [hunk 5]
```diff
@@ -1327,6 +1342,11 @@ export const MobilePortrait = () => `<div class="fd-dialog-docs-static fd-calend
 MobilePortrait.storyName = 'Mobile portrait mode with today navigation';
 MobilePortrait.parameters = {
     docs: {
-        iframeHeight: 500
+        iframeHeight: 500,
+        description: {
+            story: 'Similar to the mobile landscape example, mobile portrait mode has a Today button. To make date picker visible in portrait mode please add `fd-calendar--mobile-portrait` class.'
```
Comment (mikerodonnell89) [inline stories/datepicker/datepicker.stories.js:1347]:
```suggestion
            story: 'Similar to the mobile landscape example, mobile portrait mode has a "Today" button. To make the date picker visible in portrait mode, add `fd-calendar--mobile-portrait` class.'
```

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
Nice work! Suggested a few minor changes

**Pull request closed without merging**